File: Iniciante/1046.c
#include <stdio.h>

int main(){

    int i, f;

    scanf("%d %d", &i, &f);

    if(i==f){
        printf("O JOGO DUROU 24 HORA(S)\n");
    }else if(i>f){
        printf("O JOGO DUROU %d HORA(S)\n", 24-i+f);
    }else{
        printf("O JOGO DUROU %d HORA(S)\n", f-i);
    }

    return 0;
}

File: Iniciante/1046.php
<?php

$L = explode(" ", readline());
$i = $L[0];
$f = $L[1];

if($i==$f){
    echo "O JOGO DUROU 24 HORA(S)\n";
}elseif($i>$f){
    echo "O JOGO DUROU ".(24-$i+$f)." HORA(S)\n";
}else{
    echo "O JOGO DUROU ".($f-$i)." HORA(S)\n";
}

?>